wolfcrypt/src/chacha.c

```diff
@@ -222,7 +222,9 @@ static WC_INLINE void wc_Chacha_wordtobyte(word32 x[CHACHA_CHUNK_WORDS],
 
     for (i = 0; i < CHACHA_CHUNK_WORDS; i++) {
         x[i] = PLUS(x[i], state[i]);
+#ifdef BIG_ENDIAN_ORDER
```

Comment:
Why the wrap on big endian now? This should be a nop due the macro right?

Reply:
it has no effect, but it's warned by `cppcheck:selfAssignment`.